test/unit/plugins/ucx_mo/ucx_mo_backend_test.cpp

```diff
@@ -26,6 +26,7 @@ using namespace std;
 #ifdef USE_VRAM
 
 #include <cuda_runtime.h>
+#include <cuda.h>
```

Comment:
Doesn't this need to go behind something like HAS_CUDA? Or USE_VRAM implies HAS_CUDA?

Reply:
USE_VRAM this is what we have for CUDA in tests.
But maybe it should be unified to HAVE_CUDA

Reply:
Fixed that